test/env/deploy/cluster.yaml

```diff
@@ -218,6 +218,12 @@ spec:
           memcachedInstance: "memcached"
           postgresInstance: postgres
           listenPort: 5555
+          authProtocol: https
```

Comment:
Does it need to be specified if we have some default values?

Reply:
I mean I think that at least one e2e should be executed with default values

Reply:
Removed those defaults from this file and e2e tests files, left just one change of region to test this functionality on deployed environment.